test/CheckoutSdkTest/Payments/Links/PaymentLinksClientTest.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Checkout.Payments.Links;
```

Comment:
Please add an extra assert at `CheckoutApiTest::ShouldInstantiateAndRetrieveClientsDefault` for the new client.